Terminal.Gui/Application.cs

```diff
@@ -1227,7 +1227,7 @@ public static void OnMouseEvent (MouseEventEventArgs a)
 			}
 		}
 
-		bool FrameHandledMouseEvent (Frame frame)
+		bool FrameHandledMouseEvent (Adornment frame)
```

Comment:
`AdornmentHandledMouseEvent`

Reply:
I think you didn't pull the last changes because I didn't changes to the `Application.cs`.

Reply:
Ok I think you really want I do this change on this PR, right? Besides I think it should done on a separate PR but I'll changes in this PR.
Done.